### PR TITLE
:tada: Importing EPSS score from Snyk

### PR DESCRIPTION
**Pull request by quirinziessler**
This PR should be merged after https://github.com/DefectDojo/django-DefectDojo/pull/9516.

It adds support for importing the EPSS score from Snyk files import.

### CODE REVIEW

File: unittests/tools/test_snyk_parser.py [hunk 1]
```diff
@@ -182,3 +182,15 @@ def test_snykcode_issue_9270(self):
             parser = SnykParser()
             findings = parser.get_findings(testfile, Test())
             self.assertEqual(39, len(findings))
+
+    def test_snykcode_issue_9270_epss(self):
+        with open("unittests/scans/snyk/snykcode_issue_epss.json") as testfile:
```
Comment (manuel-sommer) [inline unittests/tools/test_snyk_parser.py:187]: `snykcode_issue_epss.json` vs `snykcontainer_issue_epss.json`

**Review by manuel-sommer — CHANGES_REQUESTED** [review]

File: unittests/tools/test_snyk_parser.py (hunk 1 above)
Comment (quirinziessler) [reply unittests/tools/test_snyk_parser.py]: Thanks for the notice!

**Review by manuel-sommer — APPROVED** [review]

**Review by devGregA — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Comment by quirinziessler** [comment]
waiting for #9516 to fix the unittests

**Comment by quirinziessler** [comment]
Sorry for this - rebase went kind of wrong

**Comment by quirinziessler** [comment]
Created #9601 as replacement for this PR.

**Pull request closed without merging**